frontend/src/app/features/personal-cabinet/components/book-details/book-details.component.scss

```
@@ -305,18 +305,58 @@
+        display: flex;
+        align-items: center;
+        gap: 6px;
+        transition: opacity 0.3s ease, transform 0.3s ease;
+        
+        &.hidden {
+            opacity: 0;
+            pointer-events: none;
+            position: absolute;
+        }
+
+        &:not(.visible):not(.hidden) {
```

Stylelint (17.5.0)

[error] Expected complex :not() pseudo-class notation (selector-not-notation)

(selector-not-notation)